console/src/app/modules/add-key-dialog/add-key-dialog.component.html

```diff
@@ -16,9 +16,9 @@
     <input cnslInput [matDatepicker]="picker" [min]="startDate" [formControl]="dateControl" />
     <mat-datepicker-toggle style="top: 0" cnslSuffix [for]="picker"></mat-datepicker-toggle>
     <mat-datepicker #picker [startAt]="startDate"></mat-datepicker>
-    <span cnslError *ngIf="dateControl?.errors?.matDatepickerMin?.min">
+    <span cnslError *ngIf="dateControl && dateControl.errors && dateControl.errors['matDatepickerMin']?.min">
```

Comment:
Can't we still use optional chaining? 

```suggestion
    <span cnslError *ngIf="dateControl?.errors && dateControl.errors['matDatepickerMin']?.min">
```

See other places, too

Reply:
<img width="916" alt="Bildschirm­foto 2023-04-17 um 10 15 23" src="https://user-images.githubusercontent.com/10165752/232425861-e29b7065-b3f9-46e1-8a91-3332f1fba089.png">
this is not accepted by the linter

Reply:
Also if you insert a question mark after dateConftrol?

Reply:
Does not do the trick.. <img width="1079" alt="Bildschirm­foto 2023-04-18 um 13 50 07" src="https://user-images.githubusercontent.com/10165752/232768241-e1969c62-2b1f-4039-925b-7620fc71cdd6.png">

Reply:
ok, sry